packages/odyssey-react/src/components/TextInput/TextInput.module.scss

```diff
@@ -81,16 +80,25 @@
   }
 }
 
-.outer {
-  position: relative;
+.prefix {
+  color: var(--PrefixTextColor);
+  padding-inline-end: var(--PaddingInline);
+}
+
+.suffix {
+  color: var(--SuffixTextColor);
+  padding-inline-start: var(--PaddingInline);
 }
 
-.indicator {
```

Comment:
What did this old `indicator` class do?

Reply:
@sunnylittle-okta previously, we were using this to house the search icon for that variant. I've removed here, since we can slot an Icon into the "prefix" slot instead.

Reply:
Got it, thanks.